ignite/src/main/java/org/hibernate/ogm/datastore/ignite/query/parsing/impl/IgnitePropertyHelper.java

```diff
@@ -86,14 +87,7 @@ public PropertyIdentifier getPropertyIdentifier(String entityType, List<String>
 //					propertyAlias = aliasResolver.createAliasForEmbedded( entityAlias, currentPropertyPath, optionalMatch );
 				}
 				else {
-					propertyEntityType = associationPropertyType.getAssociatedEntityName( getSessionFactory() );
-					currentPersister = getPersister( propertyEntityType );
-					String targetNodeType = currentPersister.getEntityKeyMetadata().getTable();
-
-					throw new NotYetImplementedException();
-//					propertyAlias = aliasResolver.createAliasForAssociation( entityAlias, currentPropertyPath, targetNodeType, optionalMatch );
-//					lastAssociationPath = new ArrayList<>( currentPropertyPath );
-//					isLastElementAssociation = true;
+					isLastElementAssociation = false;
```

Comment:
How come did you remove this part?

Reply:
this change was made to fix QueriesWithAssociationsTest.